plugins/woocommerce/client/admin/package.json

```diff
@@ -221,7 +221,7 @@
 		"webpack-cli": "5.1.x",
 		"webpack-dev-server": "4.15.x",
 		"webpack-fix-style-only-entries": "^0.6.1",
-		"webpack-rtl-plugin": "2.0.x",
+		"@automattic/webpack-rtl-plugin": "6.0.x",
```

Comment:
We might want to ban `webpack-rtl-plugin` similar to https://github.com/woocommerce/woocommerce/blob/trunk/.syncpackrc#L11-L22 to ensure consistent use of `@automattic/webpack-rtl-plugin` across the monorepo.

Reply:
Also, regarding the used version, how about adding `@automattic/webpack-rtl-plugin` to https://github.com/woocommerce/woocommerce/blob/trunk/.syncpackrc#L426-L438? That section aims to reduce the efforts needed for maintaining build-related dependencies up to date without manual work overhead.